pyiceberg/io/pyarrow.py

```diff
@@ -1342,9 +1342,8 @@ def _get_column_projection_values(
 def _task_to_record_batches(
     fs: FileSystem,
     task: FileScanTask,
-    bound_row_filter: BooleanExpression,
+    schema: Schema,
```

Comment:
is this used anywhere?

Reply:
No, that's still there from some refactoring. Thanks!